docker-compose.yml

```diff
@@ -4,6 +4,7 @@ services:
 
```

Comment:
@zqian  did you install `softwareupdate --install-rosetta` this as part of the docker desktop installation with new apple chip Mac? 

I did, but I wasn't sure if it was needed or not.